pkg/pillar/types/contenttreetypes.go

```diff
@@ -121,6 +122,7 @@ type ContentTreeStatus struct {
 	FileLocation      string // Location of filestystem
 	ObjType           string
 	NameIsURL         bool
+	FinalObjDir       string // Installation dir; may differ from verified
```

Comment:
Do we need this here or can BaseOsMgr track it separately? It isn't used by volumemgr and friends, right?

Reply:
No, it is not used by anyone except ```baseosmgr```. Let me try to get rid of this in ```ContentTreeStatus```.

Reply:
Removed.